src/daomop/storage.py

```diff
@@ -571,6 +567,32 @@ def cutout(self, cutout, return_file=False):
     def zeropoint(self):
         return float(self.header.get(ZEROPOINT_KEYWORD, 30.0))
 
+    @classmethod
+    def from_frame(cls, frame):
+        """
+        Takes frame number from an .ast file line and creates a FitsImage object.
+         
+        :param frame: frame number
+        :return: FitsImage object of the associated dataset_name and ccd
+        """
+        # assuming frame is supposed to be in the form '1111111p11'
+        x = re.match("\d{7}[A-z]\d{2}", frame)
+
+        # if there is no regex match, frame must be in the wrong format
+        if x is None:
```

Comment:
Accidentally missed that since it didn't create any errors. Originally I was using groups. Shall I just switch it to re.search so it updates when I do another pull request (whenever that may be)?